ansible/config_sonic_basedon_testbed.yml

```diff
@@ -82,6 +82,11 @@
     with_items: "{{ host_if_indexes }}"
     when: ("'host_interfaces' in vm_topo_config") and ("'tor' in vm_topo_config['dut_type'] | lower")
 
+  - name: find all vlan configurations for T0 topology
+    droplet_vlan_cfg: vm_topo_config={{ vm_topo_config }} port_alias={{ port_alias }} vlan_file={{ vlan_file|default(None) }}
```

Comment:
Suggest using more readable modern Ansible yaml syntax for defining arguments:

```
droplet_vlan_cfg:
  vm_topo_config: "{{ vm_topo_config }}"
  port_alias: "{{ port_alias }}"
  vlan_file: "{{ vlan_file|default(None) }}"
```

Reply:
Your change does not match the syntax I provided in my suggestion; you simply added line breaks; you didn't add colons or quotes. Have you tested the change? I believe you will get errors unless you use the syntax I provided.

Reply:
Yes, I verified it before posting and it did work with no errors

Reply:
That's probably because Ansible is ignoring the newlines and reading the same data. However, it is not "true" yaml, which is what I suggested. I suggested using "true" yaml because is is easier to read partly because of alignment and also because code editors will detect the syntax and color-code the keys and values.